apps/website/.vuepress/theme/layouts/Layout.vue

```diff
@@ -139,3 +144,15 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+@media (max-width: 1366px) {
```

Comment:
There are two things I'm against with this approach:

1) I do not want to add any space to this container. There are specific spaces on other parts of the site and adding space here may affect _all_ of the other things that can be put into this container.
2) we have t-shirt sizes b/c b/c we recommend that they be used for layout in the html so that apps do not have to write custom layout css. As much as possible we should instead of css for layout and spacing.